resultbrowser/src/css/default.css

```diff
@@ -595,3 +606,30 @@ table.key-value-table td.value .csep {
 .no-wrap {
     white-space: nowrap;
 }
+
+/********************************************************************************************
+ *
+ * Spinner
+ *
+ ********************************************************************************************/
+
+.loader {
+    border: 8px solid #f3f3f3;
+    border-radius: 50%;
+    border-top: 8px solid #3498db;
+    width: 24px;
+    height: 24px;
+    -webkit-animation: spin 2s linear infinite; /* Safari */
+    animation: spin 2s linear infinite;
+  }
+  
+  /* Safari */
+  @-webkit-keyframes spin {
+    0% { -webkit-transform: rotate(0deg); }
+    100% { -webkit-transform: rotate(360deg); }
+  }
+  
+  @keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+  }
```

Comment:
Please do not remove the final newline character at end-of-file.

Reply:
Done